targets/ChibiOS/_nanoCLR/nanoFramework.Hardware.Rpi/nf_hardware_rpi_native_nanoFramework_Hardware_Rpi_Pio_PioBlock.cpp

```
@@ -0,0 +1,219 @@
+{
+    PIO_TypeDef *pio = PioFromIndex(param0);
+    if (pio == nullptr || param0 < 0 || param0 > 2)
+    {
+        hr = CLR_E_INVALID_PARAMETER;
+        return -1;
+    }
+
+    // first PIO call in any flow, so un-reset here
+    PioEnsureOutOfReset(param0);
+
+    unsigned short *instr = param1.GetBuffer();
```

Clang (14.0.6)

[warning] variable 'instr' is not initialized

(cppcoreguidelines-init-variables)